lib/licenses.nix

```
@@ -332,6 +332,11 @@ in mkLicense lset) ({
     fullName = "Common Public License 1.0";
   };
 
+  csl = {
+    fullName = "Cooperative Software License";
```

```suggestion
    fullName = "Cooperative Software License";
    free = false;
```

The cooperative software license is not a free software license. I'm not sure whether we can set `redistributable`. In general, it doesn't allow redistribution, but if there is no commercial advantage, it should be fine (which I think suffices for distribution via the nixos binary caches, but I'm not a lawyer).

---

That's also how we understand it, so as far as I can find, redistribution is allowed as long as there is indeed no commercial advantage.